solutions/mint-nft/components/Mint.tsx

```diff
@@ -107,16 +128,16 @@ export const Mint: React.VFC = () => {
             that can fail. This is still the prefered method of choice to host
             in the NFT community as it is decentralized.{' '}
             <span className="underline italic">
-              This process might take up to 1 minute to complete
+              This process might take up to 1 minute to complete. Please refresh
+              the OpenSea page if you didn&apos;t see your NFT.
             </span>
           </Text>
           <section className="relative w-full pb-[20%] h-48 pb-6 mt-12">
             <Image
               className="rounded-xl"
-              src={String(asset?._url)}
+              src={asset ?? ""}
```

Comment:
Empty src will error out at runtime (linter should show it though) Will it ever be empty? If so, do something like ⬇️  this higher up

```
{ asset ? <Image /> : null }
```

Reply:
Well it shouldn't be, but I do have the typing for `asset` to be `string | null` so that might be why it is causing the typing error, let me try your solution